### ISSUE TITLE
JAWS: Insert Image (via URL) toolbar dropdown button is not labelled

### ISSUE THREAD

**Issue by Mgsy** [ckeditor/ckeditor5#11896]
## 📝 Provide detailed reproduction steps (if any)

1.  Go to https://ckeditor.com/docs/ckeditor5/latest/features/images/image-upload/images-inserting.html.
2.  Start JAWS, use CTRL+Insert+C to get the available list of dropdowns.

### ✔️ Expected result

The image toolbar dropdown is labelled properly.

### ❌ Actual result

The image toolbar dropdown is labelled as "Unlabeled 1".

![](https://user-images.githubusercontent.com/20988892/172591654-d2d1e6cf-89da-4543-859a-d96ec95e49b1.png)

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mlewand** [comment]
I checked this issue and it's not that the button as a whole doesn't have a label but only the down arrow part of split button doesn't have a label (I mean it has `aria-labelledby` but it points to an empty element).

We have two options here:

1.  We can stay with generic "insert image" label (just like it is for the main button).
2.  We can customize label for the dropdown arrow to "Insert image via URL".

I find the latter to be more appropriate here as it better describes the component that will be shown.

This issue is closely related to #11833 and should be done after #11833 is merged.

**Comment by jswiderski** [comment]
This issue has also been reported by Oracle:
- aria-labelledby attribute references element that is empty
- Element's default semantics were not overridden with role="none" or role="presentation"
- Element has no title attribute (this can probably be omitted when aria-labelledby is not empty)
